ganga/GangaCore/GPIDev/Lib/Job/Job.py

```diff
@@ -35,6 +34,11 @@
 config = GangaCore.Utility.Config.getConfig('Configuration')
 
 
+if config["repositorytype"] != "Database":
+    from GangaCore.Core.GangaRepository.SubJobXMLList import SubJobXMLList
+else:
+    from GangaCore.Core.GangaRepository.SubJobJSONList import SubJobJsonList as SubJobXMLList
+
```

Comment:
As above. If this behaviour is to appear in many places, we might need to factor it out.